closure/goog/testing/asserts.js

```diff
@@ -664,6 +665,24 @@ goog.testing.asserts.EQUALITY_PREDICATE_CANT_PROCESS = null;
 goog.testing.asserts.EQUALITY_PREDICATE_VARS_ARE_EQUAL = '';
 
 
+/**
+* @const {!Object<string, boolean>}
+ */
+var arrayTypes = goog.testing.asserts.ARRAY_TYPES = {
```

Comment:
I don't think a top-level `var` is appropriate here.  This should only be the qualified name.

It's unfortunate that we have these unqualified names at all in this file, but we shouldn't be adding to them.

Reply:
Done.